server/package.json

```diff
@@ -25,8 +26,31 @@
     "web-tree-sitter": "^0.20.8"
   },
   "scripts": {
+    "prepack": "cd .. && [ -d client/resources ] || scripts/fetch-docs.sh && cd server && cp -r ../client/resources ../client/tree-sitter-bitbake.wasm . && tsc",
     "installServer": "installServerIntoExtension ../client ./package.json ./tsconfig.json"
   },
+  "devDependencies": {
+    "@types/glob": "^8.1.0",
+    "@types/jest": "^29.5.5",
+    "@types/mocha": "^10.0.1",
```

Comment:
mocha is only used for integration tests. You'll only need jest for server unit tests here. It shouldn't be needed to compile either. Was it?

Reply:
OK, fixed.

Reply:
mocha cannot be removed. remove it will make test failure. sorry.